pages/linux/systemctl.md

```diff
@@ -18,6 +18,10 @@
 
 `systemctl enable/disable {{unit}}`
 
+- Mask/Unmask a unit, make it impossible to start it:
```

Comment:
I've never used systemctl, so this might be a stupid question, but why would one want to do this? It strikes me as odd, and possibly an unusual action (which wouldn't fit tldr's ethos).